zkevm-circuits/src/rlp_circuit_fsm.rs

```diff
@@ -2262,6 +2220,188 @@ impl<F: Field> RlpCircuitConfig<F> {
 
         debug_assert!(meta.degree() <= 9);
 
+        ///////////////////////////////////////////////////////////////////
+        /////// Rlp Decoding Table and RLP Circuit Correspondence /////////
+        ///////////////////////////////////////////////////////////////////
+
+        // RLP Decoding Table is sorted using an id = (tx_id, format, depth, access_list_idx,
+        // storage_key_idx) but bytes in the RLP circuit (rlp_table) is processed in order.
+        // Therefore, to prevent malicious injection of stack ops that don't correspond to
+        // actual bytes in the RLP circuit, lookups are added to ensure correct correspondence.
+
+        // The Init Op is not affected by sorting and stays at first position in decoding table.
+        // For this reason, its correctness and correspondence is constrained using a gate instead
+        // of lookup.
```

Comment:
why is there no lookup from RLP circuit into RLP decoding table(storing `remaining_bytes`)?